src/js/player.js

```diff
@@ -329,19 +329,13 @@ function openMiniPlayer() {
     });
 }
 /**
- * Right click on video thumbnail opens it in pop up player
- *
- * @param {string} videoThumbnail - The URL of the video thumbnail.  Used to prevent another API call.
+ * Opens the video directly in pop up player
  *
  */
-function rightClickMiniPlayer(contextImg, contextUrl){
+function clickMiniPlayer(contextUrl){
 
-  let vId = contextUrl.split('=');
-    
+  let vId = contextUrl.split('v=');
   invidiousAPI('videos', vId[1], {}, (video)=>{
```

Comment:
Use `youtubedlGetInfo()` instead of the Invidious API.  FreeTube uses this instead so that we can avoid GeoLocation issues.  While the video might play for you currently, it's possible that it wouldn't play for someone else with this method.

Reply:
Switched to youtubedlGetInfo()